fastlane/lib/fastlane/actions/install_on_device.rb

```diff
@@ -11,7 +11,7 @@ def self.run(params)
           "ios-deploy",
           params[:extra],
           "--bundle",
-          params[:ipa]
+          params[:ipa].gsub(/ /, '\ ')
```

Comment:
But it is not, it will always pass the bundle once it is needed for compilation.